packages/broker/src/plugins/operator/MaintainOperatorValueService.ts

```diff
@@ -0,0 +1,81 @@
+import { Logger } from '@streamr/utils'
+import { MaintainOperatorValueHelper } from "./MaintainOperatorValueHelper"
+import { OperatorServiceConfig } from './OperatorPlugin'
+
+const logger = new Logger(module)
+
+const CHECK_VALUE_INTERVAL = 1000 * 60 * 60 * 24 // 1 day
+
+export class MaintainOperatorValueService {
+    config: OperatorServiceConfig
+    private checkValueInterval: NodeJS.Timeout | null = null
+    private helper: MaintainOperatorValueHelper
+
+    constructor(config: OperatorServiceConfig) {
+        logger.trace('MaintainOperatorValueService created')
+        this.config = config
+        this.helper = new MaintainOperatorValueHelper(config)
+    }
+
+    // eslint-disable-next-line class-methods-use-this
+    start(penaltyLimitFraction?: bigint): void {
+        logger.info('MaintainOperatorValueService started')
+        this.checkValue(this.config.operatorContractAddress, penaltyLimitFraction)
+        this.checkValueInterval = setInterval(() => {
+            this.checkValue(this.config.operatorContractAddress, penaltyLimitFraction)
+        }, CHECK_VALUE_INTERVAL)
+    }
+
+    async checkValue(operatorContractAddress: string, penaltyLimitFraction?: bigint): Promise<void> {
+        logger.info(`Check approximate value for operator ${operatorContractAddress}`)
+
+        const { sponsorshipAddresses, approxValues, realValues } = await this.helper.getApproximatePoolValuesPerSponsorship()
+        let totalDiff = BigInt(0)
+        let totalApprox = BigInt(0)
+        const sponsorships = []
```

Comment:
Add typing information
```suggestion
        const sponsorships: { address: string, approxValue: BigNumber, realValue: BigNumber, diff: BigNumber }[] = []
```